packages/react-provider/package.json

```diff
@@ -34,6 +34,7 @@
     "@types/enzyme-adapter-react-16": "1.0.3",
     "@types/react": "16.9.42",
     "@types/react-dom": "16.9.10",
+    "@types/react-test-renderer": "^16.0.0",
```

Comment:
https://github.com/microsoft/fluentui/blob/706305303fcf03a82052c0a9e74d8aecb102885e/scripts/create-package/plopfile.ts#L166-L170

The script used for `create-package` reads & uses versions of dependencies, if this dependency absent it fails:

![image](https://user-images.githubusercontent.com/14183168/150804875-2951fd20-2e3e-4fd0-a214-d48d714f244f.png)

Opened for better suggestions.

Reply:
can we hoist this dev dependency ?

Reply:
Do you mean to move it to root `package.json`?

Reply:
we need to move all of these devDep things to single version policy, would be nice to implement this in separate PR

Reply:
I'm working on PR to hoist this to single version policy

Reply:
https://github.com/microsoft/fluentui/issues/21395